feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/TimetableItemDetailScreen.kt

```diff
@@ -88,6 +92,13 @@ fun TimetableItemDetailScreen(
         userMessageStateHolder = viewModel.userMessageStateHolder,
     )
 
+    LaunchedEffect(uiState.shouldViewBookmarkList) {
+        if (uiState.shouldViewBookmarkList) {
+            navigateToBookmarkScreen()
```

Comment:
How about using onNavigateToBookmarkScreenRequested. Because we pass events to parent composable functions 👀